diff -ru numba-0.48.0~/setup.py numba-0.48.0/setup.py
--- numba-0.48.0~/setup.py	2020-01-27 07:01:50.000000000 -0800
+++ numba-0.48.0/setup.py	2020-04-16 11:08:55.769490847 -0700
@@ -304,7 +304,7 @@
 
 build_requires = [f'numpy >={min_numpy_build_version}']
 install_requires = [
-    f'llvmlite >={min_llvmlite_version},<{max_llvmlite_version}',
+    f'llvmlite >={min_llvmlite_version}',
     f'numpy >={min_numpy_run_version}',
     'setuptools',
 ]
